### DISCUSSION TITLE
Alternative way to detect cards and fields

### DISCUSSION THREAD

**Discussion by ZloyPupsic** [AdriaGual/marvel-snap-bot#79] [Ideas]
Found a repository with an alternative detection, but it's for discord. Maybe we can try to use some parts of it for bot? It is not detecting, but has a library
https://github.com/luke-nigro/red-skull-bot

**Comment by paperc07** [comment]
I saw this as well as a couple others when I searched github for marvel snap bots lmao I was hoping to try and join and make this work differently also but so far nothing, there's also a couple steam ones out there

**Reply by paperc07** [reply]
they gave up on  this one also it looks like https://github.com/Evolusin/marvel-snap-bot
